res/controllers/Reloop-Beatmix-2-4-scripts.js

```diff
@@ -19,9 +19,9 @@ const JogFlashCriticalTime = 15; // number of seconds to quickly blink at the en
  **********************************************************************
  * User References
  * ---------------
- * Wiki/manual : http://www.mixxx.org/wiki/doku.php/reloop_beatmix_2
- * Wiki/manual : http://www.mixxx.org/wiki/doku.php/reloop_beatmix_4
- * support forum : http://mixxx.org/forums/viewtopic.php?f=7&t=8428
+ * Wiki/manual : https://github.com/mixxxdj/mixxx/wiki/reloop_beatmix_2
```

Comment:
```suggestion
 * Wiki/manual : https://github.com/mixxxdj/mixxx/wiki/Reloop-Beatmix-2
```

Reply:
Isn't this redundant to the manual link at the begining of the XML file?

Reply:
Probably, but, hey, this comes from 2016 🤷